docs/source/explanation/algorithms/classification/multi_class_classification.rst

```diff
@@ -13,19 +15,9 @@ For the supervised training we use the following algorithms components:
 - ``Loss function``: `Influence-Balanced Loss <https://arxiv.org/abs/2110.02444>`_. This is a balancing training method that helps us to solve the imbalanced data problem.
```

Comment:
Wouldn't it be always 'class incremental' since we use snapshot from ImageNet?
![image](https://user-images.githubusercontent.com/68103869/216402367-2509357f-2208-431d-a05c-c9160ebb92a9.png)

Reply:
@JihwanEom @supersoob What do you think?

Reply:
Sorry for late comment. Even though we use the imagenet ckpt at the first time training, model classes start off with a clean-slate. So it just use CrossEntropy when there's no difference between model and data classes as stated in if-statement.

Reply:
I changed it